drivers/mmc/host/sdhci.h

```diff
@@ -482,6 +482,8 @@ struct sdhci_host {
  * block count.
  */
 #define SDHCI_QUIRK2_USE_32BIT_BLK_CNT			(1<<18)
+/* Controller has inverted card detection pin */
```

Comment:
Again, the ASPEED's controller doesn't have any problems with the CD pin. It's our schematics that is queer.
Hence, "The card detection pin is inverted in the schematics" would be better.

Reply:
done